server/mqtt.go

```diff
@@ -1622,6 +1622,10 @@ func (jsa *mqttJSA) newRequestExMulti(kind, subject, cidHash string, hdrs []int,
 		r2i[reply] = i
 	}
 
+	if timeout == 0 {
```

Comment:
i.e. "no wait"

Reply:
Humm... not really. That would cause `newRequestEx` to return `fmt.Errorf("unreachable: invalid number of responses (%d)", len(responses))`, which is not what we want here...